tiledb/as_built/storage_backends.h

```diff
@@ -55,12 +55,6 @@ static constexpr bool gcs = true;
 static constexpr bool gcs = false;
 #endif  // HAVE_GCS
 
-#ifdef HAVE_HDFS
-static constexpr bool hdfs = true;
-#else
-static constexpr bool hdfs = false;
-#endif  // HAVE_HDFS
-
```

Comment:
My original thought was to hardcode `hdfs` support in the `as_built` string to `false`, but on second thought I removed it completely, because keeping it would imply that it was possible for it to be `true` if the library was built differently.